src/transformStateWithClones.js

```diff
@@ -7,7 +7,34 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const transformState = [];
```

Comment:
stansformedStates is clearer
And also stateList, stateVarsions is clear to me.
Maybe allStates.
Transform is a verb. Using verbs is pretty clear for function names when we change something. But when we talking about array we need to describe what type of values we store there